othello/apps/tournaments/views.py

```diff
@@ -0,0 +1,157 @@
+from django.contrib import messages
+from django.core.paginator import Paginator
+from django.shortcuts import get_object_or_404, redirect, render
+from django.views.generic.list import ListView
+
+from ..auth.decorators import management_only
+from .forms import TournamentCreateForm, TournamentManagementForm
+from .models import Tournament
+from .tasks import run_tournament
+
+
+class TournamentListView(ListView):
+    model = Tournament
+    template_name = "tournaments/previous.html"
+    paginate_by = 10
+    ordering = ["-start_time"]
+
+    def get_queryset(self):
+        return Tournament.objects.finished().order_by(*self.ordering)
+
+
+def detail(request, tournament_id=None):
+    t = Tournament.objects.in_progress()
+    t = t[0] if t.exists() else None
+
+    if tournament_id is not None:
+        t = get_object_or_404(Tournament, id=tournament_id)
+
+    players = sorted(t.players.all(), key=lambda x: -x.ranking) if t is not None else None
```

Comment:
```suggestion
    players = t.players.order_by("-ranking") if t is not None else None
```

There appear to be other places where you use `sorted()` like this; please change them to use `order_by()` instead.